tests/middleware/tests.py

```diff
@@ -287,6 +287,29 @@ def test_if_none_match(self):
 
     # Other tests
```

Comment:
You could create a "Content-Length header" section instead of putting in "Other tests". I had started working on this a while ago but didn't finish it. Here are the tests I'd written:
```
# Tests for the Content-Length header

def test_content_length_header_added(self):
    response = HttpResponse()
    content_length = len(response.content)
    self.assertNotIn('Content-Length', response)
    response = CommonMiddleware().process_response(HttpRequest(), response)
    self.assertEqual(int(response['Content-Length']), content_length)

def test_content_length_header_not_added(self):
    response = StreamingHttpResponse('content')
    self.assertNotIn('Content-Length', response)
    self.assertNotIn('Content-Length', CommonMiddleware().process_response(HttpRequest(), response))

def test_content_length_header_not_changed(self):
    response = HttpResponse()
    bad_content_length = len(response.content)  10
    response['Content-Length'] = bad_content_length
    response = CommonMiddleware().process_response(HttpRequest(), response)
    self.assertEqual(int(response['Content-Length']), bad_content_length)
```
Mainly I just avoided some unnecessary intermediate variables. Feel free to borrow or ignore as you like.